sale_exception/migrations/10.0.2.0.0/pre-migration.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# © 2017 Akretion, Mourad EL HADJ MIMOUNE
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate(use_env=True)
+def migrate(env, version):
+    cr = env.cr
+    # Update exception_rule for sale.order and sale.order.line:
+    cr.execute(
```

Comment:
I think it's better to use `openupgrade.logged_query` for logging the number of results and so on.

Reply:
cr.execute replaced by openupgrade.logged_query.